apps/web/src/components/user-tools/share/share-modal.tsx

```diff
@@ -70,10 +85,41 @@ export function ShareModal({
     }
   }
 
+  async function copyContentToClipboard() {
+    try {
+      if (!pathOrId) {
+        throw new Error('No path or entity id provided.')
+      }
```

Comment:
since component does not render without `pathOrId` this should be enough:
```suggestion
    if (!pathOrId) return
    try {
```